<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1261</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(24, 24, 24);
border-radius:25</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1261</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(24, 24, 24, 0);
border_ radius:25;
border: none;
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>701</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>0</y>
        <width>361</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">backgound:rgb(24, 24, 24);
border-radius:10;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../IMG/AdDancePar.png</normaloff>../IMG/AdDancePar.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_all">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Pinky Blues&quot;;
background:rgb(51, 51, 51);
border-radius:16;
</string>
      </property>
      <property name="text">
       <string>All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_list">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Pinky Blues&quot;;
background:rgb(51, 51, 51);
border-radius:16;
</string>
      </property>
      <property name="text">
       <string>List</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>891</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(29, 29, 29)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_11">
       <widget class="QLabel" name="lbl_detail_img">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QScrollArea" name="playlistList">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>209</y>
          <width>881</width>
          <height>231</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>881</width>
           <height>231</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="lbl_detail_name">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>621</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 32pt &quot;Comfortaa&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_detail_artist">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 25pt &quot;Comfortaa&quot;;
</string>
        </property>
        <property name="text">
         <string>Artist:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_detail_playcount">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 16pt &quot;Comfortaa&quot;;
</string>
        </property>
        <property name="text">
         <string>Played:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_detail_album">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>311</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 14pt &quot;Comfortaa&quot;;
</string>
        </property>
        <property name="text">
         <string>Album</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_12">
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>881</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;</string>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>881</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;</string>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>881</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;</string>
        </property>
        <property name="text">
         <string>Age:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>881</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;</string>
        </property>
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>881</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;</string>
        </property>
        <property name="text">
         <string>Favorite music:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_email">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>701</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;background:rgb(29, 29, 29);border-radius:10;border: 1px solid white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_username">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>701</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;background:rgb(29, 29, 29);border-radius:10;border: 1px solid white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_age">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>701</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;background:rgb(29, 29, 29);border-radius:10;border: 1px solid white;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_fav_music">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>280</y>
          <width>351</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Pinky Blues&quot;;background:rgb(29, 29, 29);border-radius:10;border: 1px solid white;</string>
        </property>
        <item>
         <property name="text">
          <string>Pop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Opera</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hip hop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rhythm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Soul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reggae</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Country</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Funk </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Folk </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jazz </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disco</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Classical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronic </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blues </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New age</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Christian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Traditional </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ska </string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btn_info_save">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>340</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Pinky Blues&quot;;
background:rgb(0, 0, 0);
border-radius:5</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_gender">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>250</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Pinky Blues&quot;;background:rgb(29, 29, 29);border-radius:10;border: 1px solid white;</string>
        </property>
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Famale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btn_avatar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>141</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QScrollArea" name="songList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>881</width>
          <height>391</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>881</width>
           <height>391</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="txt_search">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>461</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_search">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>0</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_info">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Pinky Blues&quot;;
background:rgb(51, 51, 51);
border-radius:16;
</string>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_duration">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>590</y>
        <width>761</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">position: absolute;
border-radius:5;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_time">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>590</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Pinky Blues&quot;;</string>
      </property>
      <property name="text">
       <string>x:xx</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_play">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>620</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">-qt-svg-fill-color: green</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../IMG/PlayIcon.png</normaloff>../IMG/PlayIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QSlider" name="slider_volume">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>670</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">position: absolute;
border-radius:5;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_volume">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>660</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../img/VolumeIcon.png</normaloff>../img/VolumeIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_curr_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>1211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(24, 24, 24);
font: 500 14pt &quot;Comfortaa&quot;;
border-radius: 10;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_curr_img">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>570</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
